### PR TITLE
Gh pages secrets redaction

### PR DESCRIPTION
**Pull request by ElijahSwiftIBM**
### :bulb: Issue Reference

**Issue:**  #6 

### :computer: What does this address?

Adds documentation to accompany PR #9 . Additionally touches up documentation with some typo fixes and references to previously undocumented functionality.

### CODE REVIEW

File: common/add_additional_secret_traits.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+# Add Additional Secret Traits
+
+Define additional "secrets" to be redacted in pyracf output.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+**Add additional secret traits**  allows pyracf to redact values beyond passwords and password phrases. When these traits are specified on input, the values will appear redacted in logger output as well as output xml and json objects. This can be specified on any "Admin" object by setting the `generate_requests_only` class attribute to a list containing the `valid_segment_traits` keys for the chosen attributes through the class constructor.
```
Comment (lcarcaramo) [inline common/add_additional_secret_traits.md:13]:
```markdown
The `add_additional_secrets` class attribute allows pyracf to redact values beyond passwords and password phrases. When the `add_additional_secrets` class attribute is given to any "Admin" object constructor, the values that correspond to the trait keys provided will be redacted in logger output as well as any retured request xml or result dictionaries.
```

File: common/add_additional_secret_traits.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+parent: Common
+---
+
+# Add Additional Secret Traits
+
+Define additional "secrets" to be redacted in pyracf output.
```
Comment (lcarcaramo) [inline common/add_additional_secret_traits.md:8]:
```markdown
Add additional secrets to be redacted in debug log output and any returned result dictionaries or request xml.
```

File: user/standard/passphrase.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+grand_parent: User Admin
+parent: Standard
+---
+
+# passphrase
```
Comment (lcarcaramo) [inline user/standard/passphrase.md:7]:
```markdown
# Passphrase
```

File: user/standard/passphrase.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+&nbsp;
+
+Change a user's **passphrase**.
+
+#### 📥 Parameters
+* `userid`<br>
+  The **z/OS userid** of the user who's **passphrase** is being changed.
+
+* `passphrase`<br>
+  The **passphrase** to assigned to the specified user.
```
Comment (lcarcaramo) [inline user/standard/passphrase.md:44]: Change `assigned` to `assign`. Same typo exists in `user/standard/password.md` as well.

Comment (ElijahSwiftIBM) [reply user/standard/passphrase.md:44]: Nice catch, typo fixed in both places.

File: user/standard/passphrase.md (hunk 3 above)
Comment (ElijahSwiftIBM) [reply user/standard/passphrase.md:7]: Typo fixed.

File: common/add_additional_secret_traits.md (hunk 2 above)
Comment (ElijahSwiftIBM) [reply common/add_additional_secret_traits.md:8]: That is more direct, text changed.

File: common/add_additional_secret_traits.md (hunk 1 above)
Comment (ElijahSwiftIBM) [reply common/add_additional_secret_traits.md:13]: This is more descriptive, text changed.

**Review by lcarcaramo — APPROVED** [review]

**Review by lcarcaramo — APPROVED** [review]

**Pull request merged**